indexing-hadoop/src/main/java/io/druid/indexer/DetermineHashedPartitionsJob.java

```diff
@@ -179,6 +179,7 @@ public boolean run()
                       new HashBasedNumberedShardSpec(
                           i,
                           numberOfShards,
+                          null,
```

Comment:
it appears that for "hashed" partition spec, partitionDimensions is completely ignored if "targetPartitionSize" was set. from documentation, it appears they should have been considered.